### PR TITLE
Add CSMH DB provisioning automation

### PR DESCRIPTION
**Pull request by maxrothman**
Also includes local version of RDS module (see commit message)

@edx/devops

### CODE REVIEW

File: playbooks/edx-east/create_csmh_db.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+#NB: to set in promoter script:
+#  multi-zone, username, password, size
+
+- name: Create new StudentModule RDS instance
+  hosts: all
+  connection: local
+  gather_facts: false
+  vars:
+    from_db:
```
Comment (feanil) [inline playbooks/edx-east/create_csmh_db.yml]: nit: Can we explicitly set these to empty strings?

**Comment by feanil** [comment]
:+1:

File: playbooks/edx-east/create_csmh_db.yml (hunk 1 above)
Comment (maxrothman) [inline playbooks/edx-east/create_csmh_db.yml]: They're currently `null`, not empty strings. If you prefer, I could set them to `null` explicitly.

Comment (feanil) [inline playbooks/edx-east/create_csmh_db.yml]: Yes, please set it to `!!null` explicitly.

**Comment by feanil** [comment]
:+1:

**Pull request merged**